packages/uswds-core/src/js/utils/focus-trap.js

```diff
@@ -6,31 +6,32 @@ const activeElement = require("./active-element");
 const FOCUSABLE =
   'a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), iframe, object, embed, [tabindex="0"], [contenteditable]';
 
-const tabHandler = (context) => {
-  const focusableElements = select(FOCUSABLE, context);
-  const firstTabStop = focusableElements[0];
-  const lastTabStop = focusableElements[focusableElements.length - 1];
+const getFocusableElements = (context) =>
+  Array.from(select(FOCUSABLE, context));
+const firstTabStop = (context) => getFocusableElements(context)[0];
+const lastTabStop = (context) => getFocusableElements(context).at(-1);
```

Comment:
One thing that might be worth double-checking is browser support and how this will be compiled. `Array#at` is a relatively new language feature (ES2022), and has pretty good [browser support](https://caniuse.com/mdn-javascript_builtins_array_at), but unclear if this will be transpiled by Babel or meets the project's browser support commitments.

Reply:
Also noting MDN has it labeled as Baseline: Widely available

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at